.github/pull_request_template.md

```diff
@@ -1,23 +1,23 @@
-**Thank you for submitting this pull request**
+**JIRA**: [link](https://www.example.com) 
 
-**JIRA**: _(please edit the JIRA link if it exists)_ 
```

Comment:
The previous version made it clear that this was optional. Do we actually want people to file JIRAs for every PR they provide?

I don't really see the template as something for us. We know the rules. The template is for the outsiders, and shouldn't scare them away.

Reply:
We should use `<!-- Comments -->` for instructions, explanation etc.